static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -84,6 +86,26 @@ define([
         var ophanUrl = 'https://dashboard.ophan.co.uk/graph/breakdown?ab=' + this.config.test.id;
         this.getElem('ophan').innerHTML = '<a href="' + ophanUrl + '"">graph</a>';
 
+        if (window.abConfigSwitches.abShowHypothesesDashboard) {
+            var hypothesis;
+            var criteria = this.config.test.audienceCriteria ? true : false;
```

Comment:
I am not sure it will lint with multiple var statements? normally you have to chain them